src/layout/dashboard/header/DashboardHeader.js

```diff
@@ -136,49 +29,35 @@ export default function DashboardHeader() {
       justifyContent="space-between"
       spacing={{ xs: 0.5 }}
     >
+      {/* Left section */}
       {isMobile ? (
         // Mobile
         <Box sx={{ display: 'flex' }}>
           <IconButton
-            onClick={onClickToggleDrawerMobile}
             sx={{
               width: 'fit-content',
               height: 'fit-content',
               p: 0,
               zIndex: 1300,
             }}
           >
-            <Hamburger
-              toggled={isDrawerMobileOpen}
-              toggle={setIsDrawerMobileOpen}
-              size={20}
-            />
+            <Hamburger toggled={isOpen} toggle={setIsOpen} size={20} />
```

Comment:
Nhất kiểu xài setState với default prop như này khó kiểm soát nó đang set cái gì. Tệ nhất cũng phải cụ thể 1 cái param

Reply:
```suggestion
            <Hamburger toggled={isOpen} toggle={setIsOpen} size={20} />
```prop: toggled với toggle là 2 thằng trong docs nó hướng dẫn á anh=

Reply:
em có thử đổi sang thằng onClick thì thấy nó chạy bth

Reply:
Thì trên doc nó viết cho đủ demo chứ đâu có đúng hết case, ví dụ nữa cũng viết kiểu này cho chỗ khác dễ ra bug mà khó check. Cái toggle nó truyền true false sẵn r thì hàm set nó ăn đc

Reply:
Thêm nữa là đang lấy setState từ th cha đó cũng phải sửa lại nãy có nói

Reply:
em có đổi lại rồi á anh